Weird. Why is `darker` linting `IPython/core/magics/script.py`?

```
Run darker -r 60625f241f298b5039cb2debc365db38aa7bb522 --check --diff . || (
--- IPython/core/magics/script.py
+++ IPython/core/magics/script.py
@@ -213,7 +213,7 @@
                     *cmd,
                     stdout=asyncio.subprocess.PIPE,
                     stderr=asyncio.subprocess.PIPE,
-                    stdin=asyncio.subprocess.PIPE
+                    stdin=asyncio.subprocess.PIPE,
                 )
             )
         except OSError as e:
Changes need auto-formatting. Run:
    darker -r 60625f241f298b5039cb2debc365db38aa7bb522
then commit and push changes to fix.
Error: Process completed with exit code 1.
```

--- a/IPython/core/magics/script.py
+++ b/IPython/core/magics/script.py
@@ -213,7 +213,7 @@ class ScriptMagics(Magics):
                     *cmd,
                     stdout=asyncio.subprocess.PIPE,
                     stderr=asyncio.subprocess.PIPE,
-                    stdin=asyncio.subprocess.PIPE
+                    stdin=asyncio.subprocess.PIPE,
                 )
             )
         except OSError as e:

